clarifai/runners/models/model_upload.py

```diff
@@ -210,7 +210,7 @@ def download_checkpoints(self):
       logger.error(f"Failed to download checkpoints for model {repo_id}")
     else:
       logger.info(f"Downloaded checkpoints for model {repo_id}")
-    return success
+    return
```

Comment:
we want to return success or not so that we can use it when trying to do here https://github.com/Clarifai/models-images/pull/29/files to fail the init build if it’s not successfull. we return None and assert.

Reply:
what do you mean? the PR you linked made it so `success` isn't used?